api/python/provisioner/commands/check.py

```diff
@@ -609,6 +609,106 @@ def _cluster_status(*, args: str) -> CheckEntry:
 
         return res
 
+    @staticmethod
+    def _storage_lvms(*, args: str) -> CheckEntry:
+        """Storage lvms check."""
+        res: CheckEntry = CheckEntry(cfg.Checks.STORAGE_LVMS.value)
+        try:
+            from cortx.utils.kvstore.pillar import PillarStorage
+            from cortx.utils.validator.v_storage import StorageV
```

Comment:
that might be done at the top of the file:
- better practice
- less duplication

Reply:
I think it was done to avoid import errors in the provisioner if `cortx.utils` is not installed on the system.
So if we can't for that reason(or any other reason) place these imports at the top of the file, we can consider using a dynamic import approach and load `cortx.utils` on the fly when it will be needed.

Reply:
used global variable to handle error and raised exception if actual check is called

Reply:
`import_error` is too generic variable name for that python module (not only cortx utils related routine is presented here).
I would suggest to use something more specific one.

Reply:
used "cortx_py_utils_import_error" as variable name